src/coreclr/vm/comtoclrcall.cpp

```diff
@@ -505,8 +505,15 @@ extern "C" UINT64 __stdcall COMToCLRWorker(Thread *pThread, ComMethodFrame* pFra
         }
     }
 
+    if (pThread->PreemptiveGCDisabled())
+    {
+        EEPOLICY_HANDLE_FATAL_ERROR_WITH_MESSAGE(
+            COR_E_EXECUTIONENGINE,
+            W("Invalid Program: attempted to call a COM method from managed code."));
+    }
+
     // Attempt to switch GC modes.  Note that this is performed manually just like in the x86 stub because
-    // we have additional checks for shutdown races, MDAs, and thread abort that are performed only when
+    // we have additional checks for shutdown races and thread abort that are performed only when
```

Comment:
```suggestion
    // we have additional checks for thread abort that are performed only when
```
Checks for shutdown that this is talking about do not exist in CoreCLR.